<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>221</green>
        <blue>218</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>221</green>
        <blue>218</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>221</green>
        <blue>218</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="passengerScroll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>731</width>
      <height>301</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>729</width>
       <height>299</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="passengerWidget" native="true">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="addPassenger">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Passenger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>237</red>
          <green>51</green>
          <blue>59</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>237</red>
          <green>51</green>
          <blue>59</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>237</red>
          <green>51</green>
          <blue>59</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>237</red>
          <green>51</green>
          <blue>59</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="elevator1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Elevator 1</string>
    </property>
    <widget class="QLabel" name="floorLabel1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Floor: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fireButton1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FIRE ALARM</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Status: Blank</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>POWER OUTAGE</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="elevator2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Elevator 2</string>
    </property>
    <widget class="QLabel" name="floorLabel2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Current Floor: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fireButton2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>FIRE ALARM</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Status: Blank</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>POWER OUTAGE</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="endSimulation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePassenger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Passenger</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logWindow">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
